includes/Core/Util/Requires_Javascript_Trait.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Trait Google\Site_Kit\Core\Util\Requires_Javascript_Trait
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Trait to display no javascript fallback message.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+trait Requires_Javascript_Trait {
+
+	/**
+	 * Display fallback message when Javascript is disabled
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string noscript HTML tag,
+	 */
+	protected function get_noscript_html() {
+
+		ob_start();
+
+		?>
+			<noscript>
+				<div class="googlesitekit-noscript notice notice-warning">
+					<div class="mdc-layout-grid">
+						<div class="mdc-layout-grid__inner">
+							<div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-12">
+								<h3 class="googlesitekit-heading-3 googlesitekit-noscript__title">
```

Comment:
This isn't really a header; it's a lot of text and too long for a header. It should also be an `<h1>` semantically since this should be the main item of note on the page. Converting this text to be a paragraph tag and a header saying something like "Please enable JavaScript" or something like that might be a bit better.

I know it's more than the ACs mentioned, but it'll be a lot more usable and accessible with that minor change.

Reply:
I agree but I was doing what was done somewhere else / Text given in IB